src/arrayMethodSort.js

```diff
@@ -3,9 +3,37 @@
 /**
  * Implement method Sort
  */
+function compareAsStrings(a, b) {
+  const stringA = `${a}`;
+  const stringB = `${b}`;
+
+  return stringA > stringB
+    ? 1
+    : stringA < stringB
+      ? -1
+      : 0;
```

Comment:
Paul suggested me to remove nested ternaries, cause they make code a bit complicated. So I think you should do it, too)

Reply:
Illia is right

Reply:
Ilya is always right!